release_notes/Service_Fabric_ReleaseNotes_90.md

```diff
@@ -0,0 +1,153 @@
+| **(Upcoming) Deprecation of support for BinaryFormatter based remoting exception serialization** | Migrate usage to Data Contract Serilization (DCS) based remoting exception. | .NET Framework team strongly recommends against using BinaryFormatter class due to security vulnerabilities and has marked it's serialization & deserielization methods as obsolete.<br>Current applications using it will continue to work as-is, but Service Fabric strongly recommends moving to using Data Contract Serilization (DCS) based remoting exception instead.<br>Service Fabric runtime will soon be stop using BinaryFormatter based remoting exception serialization by default and move to using Data Contract Serilization (DCS) based remoting exception serialization by default. |
+| **(Upcoming) Archival & the removal of older version of Service Fabric packages from the Download Center** | Migrate to more recent and supported versions of Service Fabric runtime | Service Fabric runtime will soon be archiving and removing Service Fabric runtime version 6.4 packages and older, as well as SDK version 3.3 packages and older from the package Down Center. |
+
+<br>
+
+## Current Breaking Changes
+* **Guest Executable and Container Applications**: Guest executable and container applications created or upgraded in Service Fabric clusters with runtime versions 7.1+ are incompatible with prior Service Fabric runtime versions (e.g. Service Fabric 7.0).<br/>
+    Following scenarios are impacted:<br/>
+    * An application with guest executables or containers is created or upgraded in an Service Fabric 7.1+ cluster.<br/>
+    The cluster is then downgraded to a previous Service Fabric runtime version (e.g. Service Fabric 7.0).<br/>
+    The application fails to activate.<br/>
+    * A cluster upgrade from pre-SF 7.1 version to Service Fabric 7.1+ version is in progress.<br/>
+    In parallel with the Service Fabric runtime upgrade, an application with guest executables or containers is created or upgraded.<br/>
+    The Service Fabric runtime upgrade starts rolling back (due to any reason) to the pre-SF 7.1 version.<br/>
+    The application fails to activate.<br/>
+    To avoid issues when upgrading from a pre-SF 7.1 runtime version to an Service Fabric 7.1+ runtime version, do not create or upgrade applications with guest executables or containers while the Service Fabric runtime upgrade is in progress.<br/>
```

Comment:
```suggestion
    To avoid issues when upgrading from a pre-SF 7.1 runtime version to a Service Fabric 7.1+ runtime version, do not create or upgrade applications with guest executables or containers while the Service Fabric runtime upgrade is in progress.<br/>
```

Reply:
^an Service Fabric --> a Service Fabric